global.json

```diff
@@ -1,3 +1,11 @@
 {
-  "projects": [ "./tests/src", "./tests/src/Common" ]
+  "projects": [ "./tests/src", "./tests/src/Common" ],
```

Comment:
We can delete this line after I'm pretty sure. This seems to be from our legacy build systems in .NET core